.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Claude Code Review
 
+permissions:
```

Comment:
The changes improve security by specifying minimal required permissions. Using a personal token instead of GITHUB_TOKEN is good for more flexibility, but ensure it's properly secured.

``` 
permissions:
  contents: read
  pull-requests: write
  issues: write
```